projects/spectator/tsconfig.spec.json

```diff
@@ -14,16 +14,21 @@
       "@ngneat/spectator/jest": [
         "jest/src/public_api.ts"
       ],
+      "@ngneat/spectator/vitest": [
+        "vitest/src/public_api.ts"
+      ],
       "@ngneat/spectator/internals": [
         "internals/src/public_api.ts"
       ]
     }
   },
   "files": [
-    "test/test.ts"
+    "test/test.ts",
+    "setup-vitest.ts"
```

Comment:
I really want to avoid doing this (and updating `includes`) but I can't get Vitest to pickup the `vitest/tsconfig.spec.json` instead. If I remove this line, the setup file gets completely ignored.